.eslintrc.js

```diff
@@ -9,17 +9,20 @@ module.exports = {
   ],
   'rules': {
     'curly': ['error', 'all'],
-    'lines-between-class-members': 'warn',
-    'no-else-return': 'warn',
+    'lines-between-class-members': 'off',
+    'no-alert': 'off',
```

Comment:
Do you want to allow calls to `alert()`?

Reply:
There were 4 or 6 places when this alert occurred, and I was unsure what to exchange them to. If you say, we should not allow them, we should examine those more.

Reply:
Yes, please re-check. `alert` opens an ugly modal window and usually you only want that if you have no other choice.

Reply:
As I found out, these alerts come in the special admin page (no user should see them ever), so I disabled eslint in these lines.